vuu-ui/packages/vuu-filters/src/filter-clause/ExpandoCombobox.tsx

```diff
@@ -126,6 +126,22 @@ export const ExpandoCombobox = forwardRef(function ExpandoCombobox<
     [itemToString, onSelectionChange]
   );
 
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  const getDefaultSelected = (): any => {
+    if (initialValue.current === undefined) {
+      return undefined;
+    } else if (Array.isArray(initialValue.current)) {
+      return props.source?.filter((item) =>
+        initialValue.current.includes(itemToString(item))
+      );
+    } else {
+      return props.source?.find(
+        (item) => itemToString(item) === initialValue.current
+      );
```

Comment:
I think `initialValue.current` will always be a string (as passed in `value` will always be a string). So we do not need the `Array.isArray` check.

But then we might not be handling the multi-select case. Would it be better if we get the `defaultSelected` from `ExpandoCombobox` as a prop? And then we can just create a ref in here and pass that to `Combobox`? That way we leave the decision to the consumer (who'd know whether it's a single/multi select)

Reply:
good spot, this is incorrect.

Reply:
Looking more closely, the reason the code works is because at runtime the value passed in CAN actually be an array. It's the value attribute from the Filter clause, which is a `FilterClauseValue` which can be an array.

I'm confusing everything and invalidating the type checking with this (in FilterClauseEditor)
```
value={value as string}
```
Was probably meant to be a temp hack while I got things working but forgot about it.

I'll fix the types